### ISSUE TITLE
Tonic neck reflex asymmetrical and persistent

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4794]

**Comment by pnrobinson** [comment]
@MickeySegal
I think the label should be "Persistent asymmetrical tonic neck reflex" -- is the following what you mean?
I would add it as a child of https://hpo.jax.org/app/browse/term/HP:0002476

Definition:Persistence beyond the normal age (roughly the first half of the first year of life) of the asymmetric tonic neck reflex (ATNR), which is an easily elicited primitive reflex in the immediate newborn period. The ATNR refers to the phenomenon whereby when the face of an infant is turned to one side, the ipsilateral arm and leg extend and the contralateral arm and leg flex. This posture has been compared to a typical posture of fencers.
Synonym: Persistent fencer's reflex

**Comment by MickeySegal** [comment]
Your formulation is fine, and essentially identical to ours with differences in word order.  We structure our terms to put the most specific words first because our users scan our lists of useful findings and choose ones to comment upon.  But you don't have that consideration and can choose an order that is more grammatically appropriate.

**Comment by pnrobinson** [comment]
@MickeySegal adding as HP:0032549. I added the simulconsult label as an exact synonym
